daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -587,7 +585,7 @@ optionsParser numProcessors parsePkgName = Compiler.Options
     <*> lfVersionOpt
     <*> optDebugLog
     <*> (concat <$> many optGhcCustomOptions)
-    <*> optScenarioService
+    <*> pure enableScenarioService
```

Comment:
Are we removing the option for the user to disable the scenario service? I think that's a useful option to have - e.g. on big projects it takes at extra 3Gb memory for things that are already very close to machine limits.

Reply:
No, the option was mostly a gross hack so that we could disable the scenario service even if we had an event logger, e.g., in `damlc build`. For almost all commands this option doesn’t make sense, e.g., there is no reason to enable the scenario service in `damlc build` and there is no reason to disable the scenario service in `damlc test`. The only exception to that is `damlc ide` but that one actually doesn’t use `optionsParser` so it wasn’t possible to disable the scenario service before either. I do plan to add that option (it would also allow us to run LSP tests on CI even before we have figured out why grpc sometimes breaks) but there is no reason to do it in this PR.

Reply:
Ack, that makes a lot of sense!